libbeat/docs/release-notes/breaking/breaking-8.0.asciidoc

```diff
@@ -5,11 +5,56 @@
 <titleabbrev>8.0</titleabbrev>
 ++++
 
+See the <<release-notes,release notes>> for a complete list of breaking changes,
+bug fixes, and enhancements, including changes to beta or experimental
+functionality.
+
 //NOTE: The notable-breaking-changes tagged regions are re-used in the
 //Installation and Upgrade Guide
 
 //tag::notable-breaking-changes[]
 
-Changes will be described in the GA release.
+[discrete]
+==== {beats} logs are now ECS-compliant
+
+All Elastic {beats} now write ECS-compliant logs in JSON format, and the config
+options for enabling ECS/JSON are removed. The new log name format is
+`{beatname}-{date}(-n)?.ndjson`. For example (from oldest to newest):
+`filebeat-20200101.ndjson`, `filebeat-20200101-1.ndjson`,
+`filebeat-20200101-2.ndjson`, and so on.
+
+With this change, we now provide a uniform logging scheme across Elastic
+products, which makes it easier to develop ECS-aware visualizations for
+behaviors like correlating metrics or events across products.
+
+If you have any monitoring configurations that expect the old log format, you
+must update them to use the new log format.
+
+[discrete]
+==== {beats} now store events in data streams instead of indices
+
+All {beats} shippers now store events in data streams in {es} instead of indices
+regardless of the {es} version. This change affects the underlying data storage
```

Comment:
Not sure about the regardless of the ES version part. IIRC we are only going to support writing to ES >= beats though it's not going to be enforced until 8.1. @kvch can you clarify?

Reply:
The statement is correct, we load a template with data streams regardless of the ES version. However, we refuse to connect to older ES versions from Beats 8.1. (The restriction can be turned off by setting `output.elasticsearch.allow_older_versions`)